project_resource_calendar/i18n/fr.po

```diff
@@ -126,8 +110,16 @@ msgstr "Capacité"
 #. module: project_resource_calendar
 #: model:ir.model.fields,help:project_resource_calendar.field_resource_calendar_instrument_allow_double_book
 #: model:ir.model.fields,help:project_resource_calendar.field_resource_calendar_room_allow_double_book
-msgid "Check if this resource can be booked in more than one meeting or event at the same time."
-msgstr "Vérifier si la ressource peut être réservée plusieurs fois aux même temps."
+msgid ""
+"Check if this resource can be booked in more than one meeting or event at "
+"the same time."
+msgstr "Vérifier si la ressource peut être réservée concomitamment."
+
+#. module: project_resource_calendar
+#: code:addons/project_resource_calendar/models/ir_actions_report.py:26
+#, python-format
+msgid "Choose Your Document Layout"
```

Comment:
Better to  change Choose your document layout

Reply:
It was written like this in odoo